packages/icon/src/createIconComponent.tsx

```diff
@@ -32,20 +25,15 @@ export default function createIconComponent<
   const Icon = ({ glyph, size = Size.Default, ...rest }: IconProps) => {
     const SVGComponent = glyphs[glyph];
 
-    return (
-      <SVGComponent
-        {...rest}
-        size={typeof size === 'number' ? size : sizeMap[size]}
-      />
-    );
+    return <SVGComponent {...rest} size={size} />;
```

Comment:
Should we just remove size from the arguments and just pass through all props as the default is now handled in the glyph itself?

Reply:
👍